src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+
+public class StringTest {
+    @Test
+    void split() {
+        String[] strings = "1,2".split(",");
+        assertThat(strings).contains("1");
+    }
+
+    @Test
+    void splitArray() {
+        String[] str = "1,2".split(",");
+        assertThat(str).containsExactly("1","2");
+    }
+
+    @Test
+    void substring() {
+        String str = "(1,2)";
+        String result = str.substring(1, str.length() - 1);
+        assertThat(result).contains("1,2");
+    }
+
+    @Test
+    void charAt() {
+        String string = "abc";
+        char c = string.charAt(0);
+        assertThat(c).isEqualTo('a');
```

Comment:
```suggestion
        assertThat("abc".charAt(0)).isEqualTo('a');
```
이렇게 간단한 test case의 경우 로컬 변수 없이 한 라인으로 구현해도 되지 않을까?